server/src/seeds/seed.ts

```diff
@@ -0,0 +1,27 @@
+import 'reflect-metadata';
```

Comment:
Is there a need for those? I would remove it because it doesn't add any functional value, but makes the app bigger

Reply:
I mean, the seeds, in general. It's a blockchain application, and blockchain should be the source of information